### PR TITLE
[WIP] feature/TDP-2497 Review playground navbar style

### PR DESCRIPTION
**Pull request by frassinier**
Review HTML/CSS in order to inherit from Talend Bootstrap theme

### CODE REVIEW

File: dataprep-webapp/package.json [hunk 1]
```diff
@@ -34,9 +34,11 @@
     "angular-translate": "^2.10.0",
     "angular-translate-loader-static-files": "^2.10.0",
     "angular-translate-once": "^1.0.3",
+    "angular-ui-bootstrap": "^2.1.3",
     "angular-ui-router": "^0.2.15",
     "angularjs-toaster": "^0.4.10",
     "animate.css": "^3.5.1",
+    "bootstrap-tlnd-theme": "^0.4.x",
```
Comment (frassinier) [inline dataprep-webapp/package.json]: You should not block the minor version while we are in dev

File: dataprep-webapp/src/app/components/dataset/list/dataset-list-component.spec.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ describe('DatasetList component', () => {
     let createElement;
     let element;
     let stateMock;
+    let body = angular.element('body');
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/dataset/list/dataset-list-component.spec.js]: `const` if body is not updated

File: dataprep-webapp/src/app/components/export/_export.scss [hunk 3]
```diff
@@ -10,6 +10,22 @@
...
+
+  .btn-lg {
+    border-radius: 0;
+    padding: 13px 10px 11px 10px;
+    text-transform: uppercase;
+  }
+
+  .dropdown-menu {
+    padding: 0;
+    .list-group {
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/export/_export.scss]: You should add new line

File: dataprep-webapp/src/app/components/history-control/_history.scss [hunk 4]
```diff
@@ -24,13 +24,9 @@
...
       cursor: default;
     }
-
-    &.redo {
-      margin-left: 5px;
-    }
-
-    &.undo {
-      margin-right: 5px;
+    &:hover,&:focus {
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/history-control/_history.scss]: Go back to new line and don't forget `&:active` state

File: dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss [hunk 5]
```diff
@@ -10,7 +10,39 @@
...
+    ul.inventory-actions-related-item-menu {
+      font-family: $sourcesanspro;
+      color: $black;
+
+      > li {
+        border: none;
+        font-size: 0.8em;
+        padding: 10px;
+
+        &:hover {
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss]: If you have to deal with pointer that means you miss the good HTML element :/

File: dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss [hunk 6]
```diff
@@ -10,7 +10,39 @@
...
+body {
+  > .dropdown-menu {
+    min-width: 500px;
 
+    &:before {
+      top: -1.2em;
+    }
+
+    ul.inventory-actions-related-item-menu {
+      font-family: $sourcesanspro;
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss]: Source Sans Pro is the default one by Talend/bootstrap-theme, why did you need to repeat?

File: dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss [hunk 7]
```diff
@@ -119,6 +159,18 @@
           display: none;
         }
 
+        .btn {
+          padding: 0;
+          font-size: 1.0em;
+          color: $white;
+          &:hover {
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss]: You must deal with focus state too

File: dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss [hunk 8]
```diff
@@ -136,47 +188,12 @@
...
-                }
-              }
-            }
-          }
         }
       }
     }
   }
 
+
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss]: This line is not really helpful

File: dataprep-webapp/src/app/components/playground/header/_playground-header-preview.scss [hunk 9]
```diff
@@ -0,0 +1,15 @@
+.preview {
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/playground/header/_playground-header-preview.scss]: This is not well formatted I guess

File: dataprep-webapp/src/app/components/playground/header/_playground-header-right.scss [hunk 10]
```diff
@@ -1,4 +1,4 @@
-$navigation-height: 50px;
+$navigation-height: 48px;
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/playground/header/_playground-header-right.scss]: This is default by Talend/boostrap-theme?

File: dataprep-webapp/src/app/components/widgets/navigation-list/_navigation-list.scss [hunk 11]
```diff
@@ -35,16 +35,21 @@
...
     text-align: center;
 
-    button {
+    div {
       padding: 0;
       margin: 0 5px;
       background-color: transparent;
+      vertical-align: middle;
+      width: 0;
+      height: 0;
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/widgets/navigation-list/_navigation-list.scss]: WAT?

Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/widgets/navigation-list/_navigation-list.scss]: https://css-tricks.com/snippets/css/css-triangle/

File: dataprep-webapp/src/app/components/dataset/list/dataset-list-component.spec.js (hunk 2 above)
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/dataset/list/dataset-list-component.spec.js]: ok!

File: dataprep-webapp/package.json (hunk 1 above)
Comment (charlesnguyen) [inline dataprep-webapp/package.json]: We should also rename it after "bootstrap-talend-theme"

File: dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss (hunk 6 above)
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/inventory/item/_inventory-item.scss]: This is an old style. I just moved it around. It should be removed.

File: dataprep-webapp/src/app/components/folder-selection/folder-tree-node/folder-tree-node.html [hunk 12]
```diff
@@ -18,14 +18,9 @@
      ng-class="{'folder-selected': $ctrl.isSelected}"
```
Comment (amaalej) [inline dataprep-webapp/src/app/components/folder-selection/folder-tree-node/folder-tree-node.html]: This comment is actually for line 17: 
    <pre> style="padding-left:{{($ctrl.level * 20) + $ctrl.hasChildren ? 0 : 5}}px"</pre>
As you can see, with the new caret solution, each folder in the tree is shifted randomly to the left. I propose you to do the right calculations: 
<pre>style="padding-left:{{$ctrl.getLeftPadding()}}px" </pre> and in the controller you add the getLeftPadding() function:
<pre>       
    controller: class {
        getLeftPadding() {
            return (this.level * 20) + (this.hasChildren ? 0 : 6);
        }
    },
</pre>

File: dataprep-webapp/src/app/components/dataset/list/dataset-list-component.spec.js [hunk 13]
```diff
@@ -128,12 +128,6 @@ describe('DatasetList component', () => {
         expect(element.find('.inventory-description').eq(2).text()).toBe('owned by anonymousUser, created ' + momentize('14370202197') + ', contains  lines');
     }));
 
-    it('should create related preparations list', inject(() => {
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/dataset/list/dataset-list-component.spec.js]: talend-button-dropdown is now managed by the angular-ui-bootstrap

File: dataprep-webapp/src/app/components/filter/bar/_filter-bar.scss [hunk 14]
```diff
@@ -44,6 +44,7 @@
 
     #filter-list {
       padding: 3px 10px;
+      margin: 0;
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/filter/bar/_filter-bar.scss]: I don't think so since the search input and the badge should be aligned

File: dataprep-webapp/src/app/components/widgets/navigation-list/navigation-list.html [hunk 15]
```diff
@@ -18,8 +18,8 @@
         <span ng-click="navigationListCtrl.onAddItem()" class="add-button" data-icon="s"  translate-once-title="ADD_LOOKUP_DATASET_TITLE"></span>
     </div>
     <div class="back-forth-btn">
-        <button class="arrow-left"></button>
-        <button class="arrow-right"></button>
+        <div class="arrow-left"></div>
```
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/widgets/navigation-list/navigation-list.html]: icon

the right icon is not correct. So I replace it by a div.

Comment (frassinier) [inline dataprep-webapp/src/app/components/widgets/navigation-list/navigation-list.html]: It's a `background-repeat` issue I guess

File: dataprep-webapp/src/app/components/folder-selection/folder-tree-node/folder-tree-node.html (hunk 12 above)
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/folder-selection/folder-tree-node/folder-tree-node.html]: Not work on IE11!

File: dataprep-webapp/src/app/components/widgets/navigation-list/navigation-list.html (hunk 15 above)
Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/widgets/navigation-list/navigation-list.html]: I think so but we already try to use background-repeat: no-repeat but no success
